### PR TITLE
Santiago Peralta

### PR DESCRIPTION
**Pull request by SantiagoPeralta**

### CODE REVIEW

**Comment by RecuencoJones** [comment]
Please fix the conflicts

File: description.md [hunk 1]
```diff
@@ -32,12 +32,15 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
...
-4. Next, click on *Environment Variables* button, found in *Advanced* tab. Then we want to add 2 new user variable, **GRADLE_HOME** and **PATH**.
+3a. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
+3b. You can also search in the search start main "Environment Variables".
+4a. Check to have the variable "JAVA_HOME" created, but this created, create it with the directory path of the recommended version of java.
+4b. Next, click on *Environment Variables* button, found in *Advanced* tab. Then we want to add 2 new user variable, **GRADLE_HOME** and **PATH**.
     * To add a new variable, click on *New...* button (from user variables frame). For **GRADLE_HOME** variable enter *GRADLE_HOME* into the input field for *Variable name*, and the location of the dowloaded gradle into *Variable value*. For **PATH** variable, enter *PATH* and *%GRADLE_HOME%\bin*. If **PATH** existed before we created, then we have to edit it. For that, click on *Edit...*, and add *;%GRADLE_HOME%\bin* at the end.
+    * You can avoid creating the GRADLE_HOME or JAVA_HOME variables, insert the full path in the "Path" variable.
```
Comment (RecuencoJones) [inline description.md]: Please, do not even mention this, it is definitely not a good practice

File: description.md [hunk 2]
```diff
@@ -32,12 +32,15 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
...
+#INSTALLING GRADLE ON WINDOWS 7 or Higher(WINDOWS 10)
+1. Gradle requires a JDK to be installed, version 7 or higher. Visit this [link](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk) to download and more information.(For version 3.1 of Gradle version 1.8.0.102 Java Recommends)
 2. You can now download Gradle binary only distribution [here](https://gradle.org/gradle-download/) and unpack it into a folder on your system.
-3. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
-4. Next, click on *Environment Variables* button, found in *Advanced* tab. Then we want to add 2 new user variable, **GRADLE_HOME** and **PATH**.
+3a. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
+3b. You can also search in the search start main "Environment Variables".
+4a. Check to have the variable "JAVA_HOME" created, but this created, create it with the directory path of the recommended version of java.
+4b. Next, click on *Environment Variables* button, found in *Advanced* tab. Then we want to add 2 new user variable, **GRADLE_HOME** and **PATH**.
```
Comment (RecuencoJones) [inline description.md]: Path is probably already defined, we do not want people to be misled and create another path or even overwrite it

File: description.md [hunk 3]
```diff
@@ -46,6 +49,8 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
     - If this happens, building a project will result in a failure. The most common cause is that Windows system looks an incorrect folder while searching a Java resource. To fix this, a new user variable must be created (like *GRADLE_HOME* one).
     - Name it *JAVA_HOME*, and refers to JDK installation folder (*C:\Program Files\Java\jdk1.8...* is a possible route).
     - Ultimately, edit the variable *PATH* adding *;%JAVA_HOME%\bin* at the end.
+    - If the variables are created, check that the variables named above have the updated path of the latest version, else set the path updated.
+    - You can put just type in the full path of java and Gradle in the variable "Path"
```
Comment (RecuencoJones) [inline description.md]: Same as mentioned before

File: description.md [hunk 4]
```diff
@@ -46,6 +49,8 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
     - If this happens, building a project will result in a failure. The most common cause is that Windows system looks an incorrect folder while searching a Java resource. To fix this, a new user variable must be created (like *GRADLE_HOME* one).
     - Name it *JAVA_HOME*, and refers to JDK installation folder (*C:\Program Files\Java\jdk1.8...* is a possible route).
     - Ultimately, edit the variable *PATH* adding *;%JAVA_HOME%\bin* at the end.
+    - If the variables are created, check that the variables named above have the updated path of the latest version, else set the path updated.
```
Comment (RecuencoJones) [inline description.md]: else update the path*

File: description.md [hunk 5]
```diff
@@ -32,12 +32,15 @@ If you are using gradlew, then you must also add your `<gradle/wrapper/gradle-wr
 #TRAVIS CI
 *Travis CI* is distributed continuous integration service, which supports different languages, used to build and test software projects hosted at GitHub. It allows to connect your GitHub repository and check it after each push. Its main advantage is that we could probe our libraries or applications using several configurations without installing them and that is why it uses different runtimes.60 *Travis CI* could be activated for different repositories. Moreover, it is configured by **travis.yml** file, which is situated on the root directory of each repository. This file includes information about the language, the building and testing environment and other aspects. *Travis CI* supports these languages: C, C++, C#, Clojure, D, Erlang, F#, Go, Groovy, Haskell, Java, JavaScript, Julia, Perl, PHP, Python, R, Ruby, Rust, Scala and Visual Basic.	
 
-#INSTALLING GRADLE ON WINDOWS 7
-1. Gradle requires a JDK to be installed, version 7 or higher. Visit this [link](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk) to download and more information.
+#INSTALLING GRADLE ON WINDOWS 7 or Higher(WINDOWS 10)
```
Comment (RecuencoJones) [inline description.md]: Keep it all uppercase, this works for windows 8 to, just remove the versions

File: classes/production/lab1-git-race/application.properties [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# View Resolver
```
Comment (RecuencoJones) [inline classes/production/lab1-git-race/application.properties]: Not sure about this file, but I think it was removed some contributions ago, @fjlopez can you confirm?

Comment (SAReyes) [inline classes/production/lab1-git-race/application.properties]: I believe the whole `classes` directory should be ignored, let's wait for @fjlopez to confirm it though.

File: README.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 # Web Engineering 2016-2017 / Lab1 Git Race
-[![Quality Gate](https://sonarqube.com/api/badges/gate?key=lab1-git-race)](https://sonarqube.com/dashboard/index/lab1-git-race)
+[![Quality Gate](https://sonarqube.com/api/badges/gate?key=lab1-git-race)](https://sonarqube.com/dashboard/index/lab1-git-race)
```
Comment (SAReyes) [inline README.md]: Apart from adding your entry to the students table, refrain from modifying this file please.

Update this file so it only adds your entry, at the moment you are modifying entries from other students and duplicating rows.

File: classes/production/lab1-git-race/application.properties (hunk 6 above)
Comment (fjlopez) [inline classes/production/lab1-git-race/application.properties]: Yep, the whole classes directory should be ignored.

File: description.md [hunk 8]
```diff
@@ -54,11 +54,13 @@ This tells Heroku to run the .war found in build/libs as the web app. Just in ca
...
 
-#INSTALLING GRADLE ON WINDOWS 7
-1. Gradle requires a JDK to be installed, version 7 or higher. Visit this [link](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk) to download and more information.
+#INSTALLING GRADLE ON WINDOWS 7 or Higher
+1. Gradle requires a JDK to be installed, version 7 or higher. Visit this [link](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk) to download and more information.(For version 3.1 of Gradle version 1.8.0.102 Java Recommends)
 2. You can now download Gradle binary only distribution [here](https://gradle.org/gradle-download/) and unpack it into a folder on your system.
```
Comment (fjlopez) [inline description.md]: Reorder the sentence.

File: description.md [hunk 9]
```diff
@@ -54,11 +54,13 @@ This tells Heroku to run the .war found in build/libs as the web app. Just in ca
...
-3. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
-4. Next, click on *Environment Variables* button, found in *Advanced* tab. Then we want to add 2 new user variable, **GRADLE_HOME** and **PATH**.
+3a. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
```
Comment (fjlopez) [inline description.md]: Use markdown for nested lists. See [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)

File: description.md [hunk 10]
```diff
@@ -54,11 +54,13 @@ This tells Heroku to run the .war found in build/libs as the web app. Just in ca
...
+3b. You can also search in the search start main "Environment Variables".
+4a. Check to have the variable "JAVA_HOME" created, but this created, create it with the directory path of the recommended version of java.
+4b. Next, click on *Environment Variables* button, found in *Advanced* tab. Then we want to add 2 new user variable, **GRADLE_HOME** and **PATH**.
```
Comment (fjlopez) [inline description.md]: Remove this reference to the PATH variable.

File: description.md [hunk 11]
```diff
@@ -54,11 +54,13 @@ This tells Heroku to run the .war found in build/libs as the web app. Just in ca
...
+#INSTALLING GRADLE ON WINDOWS 7 or Higher
+1. Gradle requires a JDK to be installed, version 7 or higher. Visit this [link](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk) to download and more information.(For version 3.1 of Gradle version 1.8.0.102 Java Recommends)
 2. You can now download Gradle binary only distribution [here](https://gradle.org/gradle-download/) and unpack it into a folder on your system.
-3. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
-4. Next, click on *Environment Variables* button, found in *Advanced* tab. Then we want to add 2 new user variable, **GRADLE_HOME** and **PATH**.
+3a. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
+3b. You can also search in the search start main "Environment Variables".
+4a. Check to have the variable "JAVA_HOME" created, but this created, create it with the directory path of the recommended version of java.
```
Comment (fjlopez) [inline description.md]: Use the same format convention for all mentions regarding environment variables.

File: description.md [hunk 12]
```diff
@@ -54,11 +54,13 @@ This tells Heroku to run the .war found in build/libs as the web app. Just in ca
...
 
-#INSTALLING GRADLE ON WINDOWS 7
-1. Gradle requires a JDK to be installed, version 7 or higher. Visit this [link](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk) to download and more information.
+#INSTALLING GRADLE ON WINDOWS 7 or Higher
+1. Gradle requires a JDK to be installed, version 7 or higher. Visit this [link](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html#javasejdk) to download and more information.(For version 3.1 of Gradle version 1.8.0.102 Java Recommends)
 2. You can now download Gradle binary only distribution [here](https://gradle.org/gradle-download/) and unpack it into a folder on your system.
-3. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
-4. Next, click on *Environment Variables* button, found in *Advanced* tab. Then we want to add 2 new user variable, **GRADLE_HOME** and **PATH**.
+3a. Now, right click on *My Computer* and then select *Properties"->"Advance system settings*.
+3b. You can also search in the search start main "Environment Variables".
```
Comment (fjlopez) [inline description.md]: Use italics when you are identifying elements in the Windows UI.

**Comment by fjlopez** [comment]
Your code keeps unsynced with upstream.

**Pull request merged**